src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -442,11 +443,20 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
      * @param {Error} e - API error
      * @return {void}
      */
-    fetchFeedItemsErrorCallback = (feedItems: FeedItems): void => {
+    fetchFeedItemsErrorCallback = (feedItems: FeedItems, errors: ElementsXhrError[]): void => {
+        const { onError } = this.props;
+
         this.setState({
             feedItems,
             activityFeedError: activityFeedInlineError,
         });
+
+        if (errors.length) {
+            onError(errors[0], ERROR_CODE_UNKNOWN, {
+                showNotification: true,
+                errors,
```

Comment:
Should we compile the failed activity type strings here rather than passing up the complete error objects?